devops/openshift/queue-consumers-deploy.yml

```diff
@@ -276,8 +276,10 @@ parameters:
     value: superuser-username
   - name: DB_PASSWORD_KEY
     value: superuser-password
-  - name: DATABASE_NAME_KEY
-    required: true
+  - name: API_SECRET_NAME
```

Comment:
API_SECRET_NAME is not required to be added here. `QUEUE_CONSUMERS_SECRET_NAME` can be used.